streams/graph.py

```diff
@@ -0,0 +1,136 @@
+"""Graphing utilities for EventStreams"""
+import networkx as nx
+import graphviz
```

Comment:
This seems to be a required dependency for streams now. I think it might be wiser to have it be a soft dependency.
Streams can likely be things run on clusters etc which don't really need rendering of graphics.